mlflow/projects/__init__.py

```diff
@@ -27,7 +29,8 @@
 _GIT_URI_REGEX = re.compile(r"^[^/]*:")
 # Environment variable indicating a path to a conda installation. MLflow will default to running
 # "conda" if unset
-MLFLOW_CONDA = "MLFLOW_MLFLOW_CONDA"
+MLFLOW_CONDA = "MLFLOW_CONDA"
+MLFLOW_ENTRY_POINT_MODE = "_local_run_entry_point"
```

Comment:
Add a comment on what this used for? (e.g., to communicate between two processes for some purpose)